README.md

```diff
@@ -131,6 +131,15 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [PepsiCo](https://www.pepsicojobs.com/internshipsprograms/jobs/241240?lang=en-us&previousLocale=en-US)| Purchase, New York <br> Plano, Texas | Software Development & Engineering Intern (No Sponsorship) |
 | [U.S. Bank](https://careers.usbank.com/global/en/job/2023-0020491/2024-Software-Engineering-Summer-Intern-Early-Careers) | Richfield, MN; and Atlanta, GA | 2024 Software Engineering Summer Intern |
 | Jane Street | New York, NY | [Software Engineer Internship](https://www.janestreet.com/join-jane-street/position/6834728002/) <br/> [Quantitative Trader Internship](https://www.janestreet.com/join-jane-street/position/6850721002/) <br/> [Quantitative Researcher Internship](https://www.janestreet.com/join-jane-street/position/6838982002/) |
+| [SkyWorks](https://careers.skyworksinc.com/job/Austin-Embedded-FW-Summer-Intern-2024-TX-73301/1054957600) | Austin, TX | Embedded FW Summer Intern |
```

Comment:
I feel like the Skyworks entry isn't enough of a "tech" internship for this repo. I'm guessing "FW" stands for "firmware", in which case this internship would involve writing code for hardware. The job page asks for an EE/ECE major and prefers to see some knowledge of embedded hardware peripherals, so this internship seems to be more meant for a CoE major rather than a CS major—someone who focuses on hardware rather than purely software.

Reply:
SkyWorks does appear to have an [AI Engineering internship](https://careers.skyworksinc.com/job/Hillsboro-AI-Engineering-Intern-OR-97006/1050573100/), so you could add that one instead (there might be others but I haven't looked into it)

Reply:
I let this one slide because this posting looks a lot like a low-level software engineering internship, if that makes sense.

> Responsbilities
> •    Participate in the design, development, and testing of embedded C on an upcoming ASIC
> •    Collaborate with team members to get support for your questions and bugs
> •    Take part in agile meetings to update the team on your progress and exchange ideas
> •    Create documentation and automated tests for your code
> 
> Qualifications and Educational Requirements
> •    BS/MS in EE/ECE or similar - you must be enrolled in an educational program for the duration of the internship
> •    Solid C programming skills
> •    A computational mindset with a demonstrated ability to grasp and discuss algorithms